_posts/2024-11-18-datafusion-fastest-single-node-parquet-clickbench.md

```diff
@@ -0,0 +1,347 @@
+
+
+Best in class performance on Parquet is now available to anyone. DataFusion’s
+open design lets you start quickly with a full featured Query Engine, including
+SQL, data formats, catalogs, and more, and then customize any behavior you need.
+I predict the continued emergence of new classes of data systems now that
+creators can focus the bulk of their innovation on areas such as query
+languages, system integrations, and data formats rather than trying to play
+catchup with core engine performance.
+
+ClickBench also includes results for proprietary storage formats, which require
+costly load / export steps, making them useful in fewer use cases and thus much
+less important than open formats (though the idea of use case specific formats
+is interesting[^2]).
+
+This blog post highlights some of the techniques we used to achieve this
+performance, and celebrates the teamwork involved.
+
+# Rallying The Community around Performance
+
+In July, 2024 [Mehmet Ozan Kabak], CEO of [Synnada], [called on the community to
```

Comment:
I certainly was inspired by the comment to help me focus where I spent my time reviewing PRs and helping push them through, though it is a good point that this may imply it motivated others as well, when I don't really know what did. 

Perhaps we could rephrase the motivation with something like this?

"Performance has long been a focus for DataFusion: one of the core benefits of DataFusion is its core performance, which  both excites contributors and attracts users. There seems to have been a renewed focus on performance recently, including a call in July 2024 from Mehmet ....?

Reply:
Sounds good to me

Reply:
I rephrased in 2507e67